gtda/diagrams/tests/test_features_representations.py

```diff
@@ -109,6 +109,21 @@ def test_pi_positive(pts):
         axis=1), axis=0)
     assert np.all(pi.fit_transform(diagrams) >= 0.)
 
+def test_large_pi_null_multithreaded():
+    """Test that, if one trivial diagram (all pts on the diagonal) is provided,
+    (along with a non-trivial one), then its pi is null when the input array is at
+    least 1MB and more than 1 process is used, triggering joblib's use of memmaps"""
+    X = np.linspace(0, 100, 300000)
+    pi = PersistenceImage(sigma=1, n_bins=10, n_jobs=2)
+    X = np.append(X, 1 + X[-1])
```

Comment:
What is the reason for this line?

Reply:
The np.append? Yeah I'm not sure actually, I copied one of the other tests and changed to input data to be larger and set n_jobs=2. The problem occurs without it though so I'll remove this.